R/add_theme.R

```diff
@@ -1,14 +1,21 @@
 #' Add NOAA formatting to figure or table
 #'
-#' @param x table or figures object from ggplot, base r plot, gt table, flextable, or kable extra
+#' @param x Table or figure object from ggplot, base R plot,
+#'  gt table, flextable, or kable extra
 #'
 #' @return Add the standard formatting for stock assessment reports for any
 #' figure or table. Currently, the function is able to format objects from:
-#' ggplot (ggplot2), base r plot, flextable (flextable), gt tables (gt), and kable tables (kableExtra).
+#' ggplot (ggplot2), base r plot, flextable (flextable), gt tables (gt),
+#'  and kable tables (kableExtra).
 #' @export
 #'
-#' @examples add_theme(ggplot2::ggplot(data = cars, ggplot2::aes(x = speed, y = dist)) +
-#'   ggplot2::geom_point())
+#' @examples
+#'\dontrun{
```

Comment:
in this situation, I feel like it would be OK to run, but probably doesn't matter

Reply:
Should I remove the `\dontrun{}`?

Reply:
We can keep for now